jujugui/static/gui/src/app/views/utils.js

```diff
@@ -412,13 +412,38 @@ YUI.add('juju-view-utils', function(Y) {
     return regex.test(relationId);
   };
 
-  utils.getRelationDataForService = function(db, service) {
+  /**
+     Return relation objects attached to the application
+
+     @method getRelationDataForService
+     @static
+     @param {Database} db to resolve charms/services on.
+     @param {Object} application in which we want the relations attached too
+     @param {Object} withApplication an optional parameter used to filter the
+      returning array by relations that match both endpoints
+   */
+  utils.getRelationDataForService = function(db, application, withApplication) {
```

Comment:
I did think this but this function is called by so many areas of the code base I thought it would be better to catch in a renaming branch that will require a larger QA.